packages/utils/src/schema/getDefaultFormState.ts

```diff
@@ -190,7 +197,9 @@ export function computeDefaults<T = any, S extends StrictRJSFSchema = RJSFSchema
   let schemaToCompute: S | null = null;
   let updatedRecurseList = _recurseList;
 
-  if (isObject(defaults) && isObject(schema.default)) {
+  if (isConstant(schema)) {
+    defaults = schema.const as unknown as T;
```

Comment:
Have you tested that we will always add const even when we turn off the auto-default feature?

Reply:
If you mean when the **emptyObjectFields** is set to **skipDefaults** then yes. I made a test which tests it with **skipDefaults** option.

Reply:
I see it. Just one small change to the test and I'll be satisfied